src/grammar.lalrpop

```diff
@@ -579,6 +583,76 @@ Todo: UntypedExpr = {
     }
 }
 
+Bitstring: UntypedExpr = {
```

Comment:
Is there a difference between a bitstring and a binary? Would it be better to always call it a binary?

Reply:
I renamed the types about 3 times and it's not completely cleaned up, thought I would see what you thought before doing a final pass.

The Erlang docs calls `<<1>>` a Bin which consists of a bunch of Segments. Elixir seems to go for bitstring and segment. They agree on what the names of the types in the segments are (binary and bitstring). I think bitstring is probably better because it properly encapsulates nesting, e.g.`<<1::1, <<1::1>>::bitstring>>` is in turn a bitstring. Arguments against, would probably have to rename the segments to be consistent, and BitstringSegmentOption<TypedExpr> is very long.

A binary is a specific subset of a bitstring, namely one where the number of bits is divisible by 8, so it feels confusing to use it to refer to the `<<x>>` syntax as a whole (I think?)